edubots_surveys.Rnw

```diff
@@ -1777,6 +1677,57 @@ based on the introduction of some pilot study, combined with initial opinion
 assessment on the students.
 
 
+
+%***********************************Extra************************************************
+
+Technology adoption literature often \textcolor{blue}{focuses} on users’ perceptions of the technology’s usefulness and
```

Comment:
Do you think this should go precisely here? Or maybe go to the state of the art?

Reply:
I marked this section as Extra so that I can have another look on either today or on Tuesday.  I will keep adding to the discussion.   I just felt that if any of the comments is not based on evidence, we should remove it.  I will keep working on it.

Reply:
Hello JJ, this is not the final version of the paper.  I will add the text in the section marked as 'Extra' once I go over the rest of the paper and see where these bits fit best.  We won't annoy the reviewer at all.  We will demonstrate to them that we thought their comments through and used their comments wisely.  That is why I am planning to have another go this sunday to check things and possibly one last check before submission.
Thank you

Reply:
Did you finally have another look?

Reply:
Hello JJ,
I think you didn't add these bits so I didn't review them!